Helpers/opent2t-translator-helper-hue/js/tests/README.md

```diff
@@ -0,0 +1,24 @@
+# Running Tests
```

Comment:
since this is going away, don't worry about the readme

---
In reply to: [81488209](https://github.com/openT2T/translators/pull/96#discussion_r81488209) [](ancestors = 81488209)